dvzo/settings.py

```diff
@@ -132,6 +133,36 @@
 
 LOCALE_PATHS = [BASE_DIR / 'templates/locale']
 
+TINYMCE_JS_URL = 'https://cdn.tiny.cloud/1/mzzbdds45pytc0srczsext77st8du79k6rp4840guy0jdayk/tinymce/5/tinymce.min.js'
+TINYMCE_COMPRESSOR = False
+TINYMCE_DEFAULT_CONFIG = {
+    "height": "320px",
+    "menubar": "file edit view insert format tools table help",
+    "plugins": "advlist autolink lists link charmap preview anchor searchreplace visualblocks code "
+               "fullscreen insertdatetime template table paste code help wordcount spellchecker",
+    "toolbar": "undo redo | bold italic underline strikethrough | fontsizeselect formatselect | alignleft "
+               "aligncenter alignright alignjustify | outdent indent | table | numlist bullist checklist | forecolor "
+               "backcolor casechange permanentpen formatpainter removeformat | pagebreak | charmap emoticons | "
+               "fullscreen | template link anchor | "
+               "code",
+    "custom_undo_redo_levels": 10,
+    "templates": [
+        {
+            "title": "Abstellplanung",
+            "description": "Fügt eine Tabelle für die Abstellplanung ein.",
+            "content": "<h3>Parkordnung</h3><p>&nbsp;</p><table class='table table-striped table-sm'><tbody><tr>"
```

Comment:
Müsste der Content nicht falls existent von dort geladen werden und falls nicht dynamisch generiert werden?
Und weshalb befindet sich das in den Settings?

Reply:
Durch die Art wie tinymce integriert ist (django-tinymce app) und wie ich das ganze verstehe, sah ich keine andere möglichkeit.
Es geht hier um "Snippets" die wiederverwendbar sind...

Reply:
Ich würde das vielleicht nochmals zusammen anschauen bevor wir das in Produktion geben. Ich kann mir fast nicht vorstellen das dies der einzige Weg ist.